django/db/backends/sqlite3/schema.py

```diff
@@ -55,8 +55,18 @@ def quote_value(self, value):
         else:
             raise ValueError("Cannot quote parameter value %r of type %s" % (value, type(value)))
 
+    def _is_referenced(self, table_name, column_name=None):
```

Comment:
I didn't immediately know the purpose of the method by the name. Maybe use a name like `_is_referenced_by_fk_constraint` or add a brief docstring?